packages/css-variables-language-server/src/CSSVariableManager.ts

```diff
@@ -66,6 +65,54 @@ const getAST = (filePath: string, content: string) => {
 export default class CSSVariableManager {
   private cacheManager = new CacheManager<CSSVariable>();
 
+  private resolveCachedVariables = () => {
+    for (const filePath of this.cacheManager.getFiles()) {
+      this.cacheManager.getAll(filePath).forEach((variable, key) => {
+        this.setCssVariable(key, this.resolveRecursiveVariables(variable.symbol.value), filePath, variable.definition.range);
+      });
+    }
+  };
+
+  public resolveRecursiveVariables = (value: string) => {
+    for (let i = 0; i < 20; i++) {
```

Comment:
@andrewda 

Why does the for loop run 20? Is it because the maximum recursion depth is thought to be 20?